.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Build
+
+on: push
+
+jobs:
+  build-module:
+    strategy:
+      matrix:
+        include:
+          - os: macos-latest
```

Comment:
No `arm64` build?

Reply:
I think eliot marked darwin/amd64 as higher prio so none for now

Reply:
Oh wait I just understood your question

Reply:
I think macos-latest is arm64
<img width="2136" height="676" alt="image" src="https://github.com/user-attachments/assets/4c4672e8-830c-4f77-ab3e-e1be6517ab6e" />